aea/helpers/yaml_utils.py

```diff
@@ -0,0 +1,153 @@
+# -*- coding: utf-8 -*-
```

Comment:
this module contains pieces of code both from aea.helpers.base and aea.configurations.loader, nothing really new.